search/grammars/filter.grammar

```diff
@@ -5,13 +5,65 @@ en,by_content_type => $ContentType about the $Text
 en,by_content_type => $ContentType of the $Text
 en,by_content_type => $Text $ContentType
 
+en,by_source => $Source $Text
```

Comment:
What we should expect from by_source?

some words from "Shamati"? is that the query?

What is the expected search result?

Reply:
Search $Term with "filter"?

Reply:
It could be words in "Shamati" and words in content units that connected to "Shamati" (excepted results are content that connected to the source).
We search for the term with a source filter.